WordPress/Sniffs/VIP/FileSystemWritesDisallowSniff.php

```diff
@@ -102,4 +118,35 @@ public function getGroups() {
 		return $groups;
 	}
 
+	/**
+	 * This sniff is active, but it's deprecated, handle the deprecation notices
```

Comment:
Not a blocker from my point of view, but a better function description would be welcome. You can re-use the default `Processes this test, when one of its tokens is encountered.` if you like or alternatively something like `Process the token and handle the deprecation notices`.